src/IO/Resources/ClilocLoader.cs

```diff
@@ -197,157 +217,165 @@ public string Translate(int clilocNum, string arg = "", bool capitalize = false)
                 return null;
             }
 
-            List<string> arguments = new List<string>();
-
             if (arg == null)
             {
                 arg = "";
             }
 
-            while (arg.Length != 0 && arg[0] == '\t')
-            {
-                arg = arg.Remove(0, 1);
-            }
+            var roChars = arg.AsSpan();
+
 
-            for (int i = 0; i < arg.Length; i++)
+            // get count of valid args
+            int i = 0;
+            int totalArgs = 0;
+            int trueStart = -1;
+
+            for (; i < roChars.Length; ++i)
             {
-                if (arg[i] == '\t')
+                if (roChars[i] != '\t')
+                {
+                    if (trueStart == -1)
+                    {
+                        trueStart = i;
+                    }
+                }
+                else if (trueStart >= 0)
                 {
-                    arguments.Add(arg.Substring(0, i));
-                    arg = arg.Substring(i + 1);
-                    i = 0;
+                    ++totalArgs;
                 }
             }
 
-            bool has_arguments = arguments.Count != 0;
+            if (trueStart == -1)
+            {
+                trueStart = 0;
+            }
 
-            arguments.Add(arg);
+            // store index locations
+            Point* locations = stackalloc Point[++totalArgs];
```

Comment:
You brought in `Span/Memory`, so I think this can be changed to `Span<Point>` and the unsafe dropped?